usp/fetch_parse.py

```diff
@@ -181,6 +188,9 @@ def sitemap(self) -> AbstractSitemap:
             )
             fetched_sitemap = fetcher.sitemap()
             sub_sitemaps.append(fetched_sitemap)
+            if crawl_interval > 0 and len(sitemap_urls.keys()) > 1:
+                log.info("Fetching next sitemap URL in {} seconds...".format(crawl_interval))
+                time.sleep(crawl_interval)
```

Comment:
You probably might want to pass the `Crawl-Delay` to recursive fetchers so that sub-sitemap fetches get delayed too.